### DISCUSSION TITLE
インストールしたテーマの保存場所はどこですか？

### DISCUSSION THREAD

**Discussion by Bowgenun** [t1m0thyj/WinDynamicDesktop#622] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/5440711b-af42-4f64-a6b5-b6500cda63af)
![image](https://github.com/user-attachments/assets/ff35bdf9-3c37-4b6f-9be5-56cd50770600)

WinDynamicDesktopからインストールした「Big Sur」、「Monterey」と言ったMacOS背景のテーマのフォルダが見つかりません。
先日はC:\Users\ユーザー名\AppData\Local\Microsoft\Windows\Themesにあるのを確認出来ましたが、その翌朝もう一度確認すると、昨日はあった筈のテーマがありません。

WinDynamicDesktopの使用経歴や環境など：
・2020年頃にGitHub経由でWinDynamicを入手した
・同時期にWinDynamic経由でMacOS背景を入手した
・2022年頃に何らかの理由でMicroSoftStore経由で新しくWinDynamicをインストールした

・2024年11月20日、MicrosoftStoreからインストールしていたWinDynamicをアンインストールした
・同日、GitHubから5.6.0版をインストールした
・C:\Users\ユーザー名\AppData\Local\Microsoft\Windows\ThemesにあるテーマのMacOS背景のjpegファイルを編集した
 jsonファイルではなく画像ファイルのみ ) 
・WinDynamicの「テーマを更新する」を使用し、編集した内容通りの画像が表示される事を確認した
・2024年11月21日、日付が変わってもテーマがシャッフルされない事に気付く
・同日、GitHubから5.50をインストールする
・同日、GitHubから5.40をインストールする
・同日、GitHubから5.21をインストールする
・毎日テーマをシャッフルする機能を取り戻す
・上述のテーマフォルダが見つからない事に気付く

以上です。

依然としてWinDynamicDesktopでは上述のテーマは使用可能なので、テーマフォルダ自体はどこかにある筈だと思うのですが、実際にフォルダが見つからないと不安で仕方がありません。
テーマフォルダの格納場所について教えて頂けますか？

**Comment by t1m0thyj** [comment]
See [Finding where app is installed](https://github.com/t1m0thyj/WinDynamicDesktop/wiki/Troubleshooting#finding-where-app-is-installed). If you have installed WDD from the Microsoft Store, the location is likely `%LOCALAPPDATA%\Packages\38719TimothyJohnson.WinDynamicDesktop_rfarkse4rn21a\LocalState\themes`

**Reply by Bowgenun** [reply, marked as answer]
ありがとうございます。あなたの意見は正しく、
C:\Users\ユーザー名\AppData\Local\WinDynamicDesktop
の場所にありました。

悔やむのは、あなたの返信を見る前に自己解決してしまった、と言う点です。
でもありがとう、あなたは正しかった。
